### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Expertium**

### CODE REVIEW

**Comment by Expertium** [comment]
@L-M-Sherlock @user1823 any suggestions and crtitique are welcome

**Comment by L-M-Sherlock** [comment]
I guess you misunderstand my words: https://github.com/open-spaced-repetition/fsrs4anki/pull/494#issuecomment-1762992968

I means, we need to notify the users. But the notification should be concise. We can put a link of the new tutorial in the notification in the README, like https://github.com/open-spaced-repetition/fsrs4anki/pull/494#issuecomment-1763276719.

**Comment by Expertium** [comment]
I thought you wanted to modify the existing tutorial. Ok, then create a new file and add a link to it to the old readme, and I'll copy the text from this PR and submit a new PR.

**Comment by L-M-Sherlock** [comment]
You can still work on the current PR.

**Comment by Expertium** [comment]
The link from README to README2 doesn't work right now, but I'm sure that will be an easy fix for you.

File: README.md [hunk 1]
```diff
@@ -36,15 +36,18 @@ _✨ A modern Anki [custom scheduling](https://faqs.ankiweb.net/the-2021-schedul
 
 # Introduction
 
-FSRS4Anki consists of two main parts: the scheduler and the optimizer.
-
-- The scheduler replaces Anki's built-in scheduler and schedules the cards according to the FSRS algorithm.
-- The optimizer uses machine learning to learn your memory patterns and finds parameters that best fit your review history. For details about the working of the optimizer, please read [the mechanism of optimization](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-mechanism-of-optimization).
+FSRS has been integrated into Anki. If your version of Anki is 23.10 or newer, use the built-in FSRS. Read about how to use it [here](https://github.com/open-spaced-repetition/fsrs4anki/README2.md).
+If your version of Anki is 2.1.66 or older, use the standalone version.
```
Comment (user1823) [inline README.md:40]: The introduction section serves as an introduction to FSRS in general. It is not specific to the standalone version. So, I think that this message should be added to the "How to Get Started?" section.

File: README.md [hunk 2]
```diff
@@ -36,15 +36,18 @@ _✨ A modern Anki [custom scheduling](https://faqs.ankiweb.net/the-2021-schedul
...
 
 For details about the FSRS algorithm, please read [the algorithm](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm). If you are interested, you can also read my papers:
 - [A Stochastic Shortest Path Algorithm for Optimizing Spaced Repetition Scheduling](https://www.maimemo.com/paper/) (free access), and
 - [Optimizing Spaced Repetition Schedule by Capturing the Dynamics of Memory](https://www.researchgate.net/publication/369045947_Optimizing_Spaced_Repetition_Schedule_by_Capturing_the_Dynamics_of_Memory) (submit a request).
 
+FSRS4Anki consists of two main parts: the scheduler and the optimizer.
+
+- The scheduler replaces Anki's built-in scheduler and schedules the cards according to the FSRS algorithm.
+- The optimizer uses machine learning to learn your memory patterns and finds parameters that best fit your review history. For details about the working of the optimizer, please read [the mechanism of optimization](https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-mechanism-of-optimization).
+
```
Comment (user1823) [inline README.md:50]: In my opinion, this part should be moved back to its original place. FSRS still consists of two main parts. It is just that they have been integrated into the same section of Deck Options.

File: README.md (hunk 1 above)
Comment (user1823) [reply README.md:40]: Also, I think that my version of the update notification is more concise: https://github.com/open-spaced-repetition/fsrs4anki/pull/494#issuecomment-1763276719

**Comment by user1823** [comment]
I have several suggestions for the README2, but I am not sure what is the best way to put them forward.

File: README.md (hunk 2 above)
Comment (Expertium) [reply README.md:50]: But it is in it's original place, in "Introduction".

**Comment by L-M-Sherlock** [comment]
Is it ready to merge? For above suggestion, we can improve the readme in the next PR.

**Comment by Expertium** [comment]
I'd say it's ready.

**Comment by user1823** [comment]
So, we can merge it. I can make further edits in a subsequent PR.

**Review by L-M-Sherlock — APPROVED** [review]

**Pull request merged**